ffmpeg.c

```diff
@@ -1513,6 +1518,7 @@ static int reap_filters(int flush)
                     break;
                 }
                 do_audio_out(of, ost, filtered_frame);
+                do_video_out(of, ost, filtered_frame, float_pts, 1);
```

Comment:
I'm not exactly sure what `float_pts` is, would probably have to be converted into what `do_video_out` expects.